.gitignore

```diff
@@ -1,2 +1,3 @@
 .metadata
 *.swp
+*.iml
```

Comment:
Are you using IntelliJ successfully with this project? That's neat!

Can you try to import all Eclipse formatting settings into IntelliJ. They are located in the `misc` folder in this repo.